packages/block-library/src/navigation/style.scss

```diff
@@ -329,7 +334,11 @@
 	bottom: 0;
 
 	.wp-block-navigation__responsive-container-content {
-		display: contents;
+		display: flex;
+		flex-wrap: var(--layout-wrap, wrap);
+		flex-direction: var(--layout-direction, initial);
+		justify-content: var(--layout-justify, initial);
```

Comment:
In a quick test, the following seems to have fixed it for me:

```suggestion
		justify-content: flex-start;
```

Can you think of any reason this shouldn't work?

Reply:
This fixes left, center, right:
<img width="756" alt="Screenshot 2021-11-08 at 08 58 16" src="https://user-images.githubusercontent.com/1204802/140704678-ef3632d7-9bba-4fbe-b41d-3317ddfbdf81.png">

<img width="764" alt="Screenshot 2021-11-08 at 08 58 27" src="https://user-images.githubusercontent.com/1204802/140704686-aef0bce3-71a9-46f8-a504-24bf44380462.png">
<img width="767" alt="Screenshot 2021-11-08 at 08 59 56" src="https://user-images.githubusercontent.com/1204802/140704715-cfde6b45-0292-4ad9-9557-7dd295450e3e.png">

<img width="770" alt="Screenshot 2021-11-08 at 08 58 38" src="https://user-images.githubusercontent.com/1204802/140704720-c207c558-8d81-45ae-bf37-93f9ea4cd689.png">
<img width="703" alt="Screenshot 2021-11-08 at 09 00 11" src="https://user-images.githubusercontent.com/1204802/140704734-abde4576-1c7f-4682-8f8b-e9a7088b9b61.png">


<img width="771" alt="Screenshot 2021-11-08 at 08 58 51" src="https://user-images.githubusercontent.com/1204802/140704743-5c01475e-b7b7-4064-a7c2-ebcfdb6f59cd.png">

Space between gets weird, I'll look at what I can do there:
<img width="769" alt="Screenshot 2021-11-08 at 09 00 38" src="https://user-images.githubusercontent.com/1204802/140704807-f1077550-93a6-4505-91d8-ab3e58105ccf.png">